docs/user/getting_started.rst

```diff
@@ -86,11 +86,19 @@ Next go back to the top-level Gluon directory and build Gluon::
     make update                        # Get other repositories used by Gluon
     make GLUON_TARGET=ar71xx-generic   # Build Gluon
 
-In case of errors read the messages carefully and try to fix the stated issues (e.g. install tools not available yet).
+In case of errors read the messages carefully and try to fix the stated issues
+(e.g. install missing tools not available or look at the wiki:
+[Troubleshooting](https://github.com/freifunk-gluon/gluon/wiki/Troubleshooting))
 
 ``ar71xx-generic`` is the most common target and will generate images for most of the supported hardware.
 To see a complete list of supported targets, call ``make`` without setting ``GLUON_TARGET``.
 
+To build all (currently 19) targets use a loop like this:
```

Comment:
Nope, we're not going to maintain that list in the documentation. Please can write their own scripts using the output of `make list-targets` though.

Reply:
good point! fixed